src/components/navbar/NavBar.js

```diff
@@ -1,26 +1,29 @@
 // @flow
-import React from 'react';
+import React, { Fragment } from 'react';
 import { connect } from 'react-redux';
 import classNames from 'classnames';
 import { MdCloudUpload, MdImage } from 'react-icons/md';
 import { NavLink } from 'react-router-dom';
 import Search from './Search';
+import Dropdown from './Dropdown';
 import TeamInitialsButton from '../team/TeamInitialsButton';
 import PresentationSwitch from './PresentationSwitch';
 import type { Team } from '../../models/Team';
 import type { AppState } from '../../state/AppState';
 import { openTeamSidebar } from '../../state/team_sidebar/teamSidebar.actionCreators';
-
 import './NavBar.scss';
 
 type Props = {
   activeTeam: ?Team,
   isTeamSidebarOpen: boolean,
+  loggedIn: boolean,
```

Comment:
I would use a name like `isLoggedIn` for consistency. The prop just before it already has this wording and using different ones is rather confusing